src/veh_interact.cpp

```diff
@@ -620,6 +619,20 @@ bool veh_interact::can_install_part(int msg_width){
                         steering_string.c_str(),
                         tire_string.c_str());
         wrefresh (w_msg);
+    } else if (is_screwable){
+        werase (w_msg);
+        fold_and_print(w_msg, 0, 1, msg_width - 2, c_ltgray,
+                        _("Needs <color_%1$s>%2$s</color>, a <color_%3$s>screwdriver</color> or <color_%5$s>duct tape</color> and level <color_%4$s>%5$d</color> skill in mechanics.%6$s%7$s%8$s"),
```

Comment:
I have some doubts about correctness of string order (%3$s, %__5__$s, %__4__$s). Can you double check that please?